docker-compose.yml

```diff
@@ -8,3 +8,11 @@ services:
       - KEYCLOAK_PASSWORD=admin
     ports:
       - 8080:8080
+
+  # TODO: use tagged image of mariadb from dockerhub
+  mariadb:
+    image: testmaria:latest
```

Comment:
as discussed, we need to merge mariaDB test image and then build a version for it. That should be used here